### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## [Unreleased]
+
+## [8.2.1] 2020-03-20
+
+### Added
+
+- First release.
+
+[Unreleased]: https://github.com/giantswarm/operatorkit/compare/v8.2.1...HEAD
```
Comment (tfussell) [inline CHANGELOG.md]: Wrong link here maybe

Comment (xh3b4sd) [reply CHANGELOG.md]: Not maybe buddy. Thanks for catching that.

Comment (xh3b4sd) [inline CHANGELOG.md]:
```suggestion
[Unreleased]: https://github.com/giantswarm/aws-operator/compare/v8.2.1...HEAD
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+[8.2.1]: https://github.com/giantswarm/operatorkit/releases/tag/v8.2.1
```
Comment (xh3b4sd) [inline CHANGELOG.md]:
```suggestion
[8.2.1]: https://github.com/giantswarm/aws-operator/releases/tag/v8.2.1
```

**Comment by sslavic** [comment]
@xh3b4sd please consider requesting review from mostly component owner where it is known https://github.com/giantswarm/giantswarm/blob/master/teams/team-ownership.md

**Comment by xh3b4sd** [comment]
Will do. Firecracker owns this thing here and I am the only PE right now so all is well.

**Pull request merged**